### ISSUE TITLE
make nocairo fails to build because of missing cairo deps

### ISSUE THREAD

**Issue by reyjrar** [bookingcom/carbonapi#133]
Attempting to compile on CentOS 6 with libcairo 1.8.8 fails, so I wanted to disable cairo since I don't need carbonapi to serve PNG's anyways.

```
+ make nocairo
go build  -ldflags '-X main.BuildVersion=eb5501d'  github.com/bookingcom/carbonapi/cmd/carbonapi
# github.com/bookingcom/carbonapi/vendor/github.com/evmar/gocairo/cairo
vendor/github.com/evmar/gocairo/cairo/cairo.go:630:32: could not determine kind of name for C.CAIRO_ANTIALIAS_BEST
vendor/github.com/evmar/gocairo/cairo/cairo.go:628:32: could not determine kind of name for C.CAIRO_ANTIALIAS_FAST
vendor/github.com/evmar/gocairo/cairo/cairo.go:629:32: could not determine kind of name for C.CAIRO_ANTIALIAS_GOOD
vendor/github.com/evmar/gocairo/cairo/cairo.go:2346:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_COGL
vendor/github.com/evmar/gocairo/cairo/cairo.go:2340:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_DRM
vendor/github.com/evmar/gocairo/cairo/cairo.go:2341:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_GL
vendor/github.com/evmar/gocairo/cairo/cairo.go:2348:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_INVALID
vendor/github.com/evmar/gocairo/cairo/cairo.go:2342:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_SCRIPT
vendor/github.com/evmar/gocairo/cairo/cairo.go:2347:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_WIN32
vendor/github.com/evmar/gocairo/cairo/cairo.go:2343:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_XCB
vendor/github.com/evmar/gocairo/cairo/cairo.go:2344:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_XLIB
vendor/github.com/evmar/gocairo/cairo/cairo.go:2345:33: could not determine kind of name for C.CAIRO_DEVICE_TYPE_XML
vendor/github.com/evmar/gocairo/cairo/cairo.go:352:26: could not determine kind of name for C.CAIRO_FORMAT_INVALID
vendor/github.com/evmar/gocairo/cairo/cairo.go:358:26: could not determine kind of name for C.CAIRO_FORMAT_RGB30
vendor/github.com/evmar/gocairo/cairo/cairo.go:481:35: could not determine kind of name for C.CAIRO_OPERATOR_COLOR_BURN
vendor/github.com/evmar/gocairo/cairo/cairo.go:480:35: could not determine kind of name for C.CAIRO_OPERATOR_COLOR_DODGE
vendor/github.com/evmar/gocairo/cairo/cairo.go:478:35: could not determine kind of name for C.CAIRO_OPERATOR_DARKEN
vendor/github.com/evmar/gocairo/cairo/cairo.go:484:35: could not determine kind of name for C.CAIRO_OPERATOR_DIFFERENCE
vendor/github.com/evmar/gocairo/cairo/cairo.go:485:35: could not determine kind of name for C.CAIRO_OPERATOR_EXCLUSION
vendor/github.com/evmar/gocairo/cairo/cairo.go:482:35: could not determine kind of name for C.CAIRO_OPERATOR_HARD_LIGHT
vendor/github.com/evmar/gocairo/cairo/cairo.go:488:35: could not determine kind of name for C.CAIRO_OPERATOR_HSL_COLOR
vendor/github.com/evmar/gocairo/cairo/cairo.go:486:35: could not determine kind of name for C.CAIRO_OPERATOR_HSL_HUE
vendor/github.com/evmar/gocairo/cairo/cairo.go:489:35: could not determine kind of name for C.CAIRO_OPERATOR_HSL_LUMINOSITY
vendor/github.com/evmar/gocairo/cairo/cairo.go:487:35: could not determine kind of name for C.CAIRO_OPERATOR_HSL_SATURATION
vendor/github.com/evmar/gocairo/cairo/cairo.go:479:35: could not determine kind of name for C.CAIRO_OPERATOR_LIGHTEN
vendor/github.com/evmar/gocairo/cairo/cairo.go:475:35: could not determine kind of name for C.CAIRO_OPERATOR_MULTIPLY
vendor/github.com/evmar/gocairo/cairo/cairo.go:477:35: could not determine kind of name for C.CAIRO_OPERATOR_OVERLAY
```

**Comment by reyjrar** [comment]
I'm not a golang dev, pretty green to all of it. I think there was a missing "// +build cairo" on expr/functions/cairo/png/pixel_ratio.go.  That's fixed in #134, but I'm not 100% sure that's all that's necessary.

**Comment by grzkv** [comment]
Hi @reyjrar Thanks for the feedback. We will try to reproduce this problem.

**Comment by blind-oracle** [comment]
@reyjrar See your PR for the fix

**Comment by grzkv** [comment]
Fixed by #134
